test/utils/toISOMonthString_spec.js

```diff
@@ -1,9 +1,11 @@
 import moment from 'moment';
 import { expect } from 'chai';
 
-import { ISO_FORMAT, ISO_MONTH_FORMAT } from '../../src/constants';
```

Comment:
why inline this, if the constant is still going to exist?

Reply:
It is only used in one test file now, and in no production code, so I don't think it makes sense to ship this code to clients.

Reply:
sure but this PR doesn’t delete the constant (which would be breaking change). Can the inlining wait til then?

Reply:
Oops, I forgot to save that file. Yeah, I can keep it for now. I'll add a note to delete it as part of the next breaking change.